src/qibolab/instruments/qblox/cluster_qcm_bb.py

```diff
@@ -141,18 +140,17 @@ def __init__(self, name: str, address: str, cluster: Cluster = None):
         self._used_sequencers_numbers: list[int] = []
         self._unused_sequencers_numbers: list[int] = []
 
-    def connect(self):
+    def connect(self, cluster: QbloxCluster = None):
```

Comment:
In principle, this is slightly wrong: `ClusterQCM_BB` is inheriting from `Instrument`, which declares a certain interface for `.connect()`:

https://github.com/qiboteam/qibolab/blob/735e34fb706ce5d10fa45a150721da5b4ce805d4/src/qibolab/instruments/abstract.py#L37-L38

and (of course) there is no `QbloxCluster` parameter in it.

However, the cluster modules are never used directly, but always through the controller. So, they do not have to be even `Instrument`s at all.

For the time being, just ignore the issue. We will reconsider during #686 and related.